contracts/interfaces/ILoanRouter.sol

```diff
@@ -7,6 +7,41 @@ import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
  * @dev Router for creating loans with tranche
  */
 interface ILoanRouter {
+    /**
+     * @dev Borrow against a given bond, wrapping the raw collateral into a ButtonToken first
+     * @param amount The amount of collateral to deposit into the bond
+     * @param currency The asset to borrow
+     * @param sales The number of tranche tokens to sell, in tranche index order
+     * @param minOutput The minimum amount of currency to get out, reverts if not met
+     */
+    function wrapAndBorrow(
+        uint256 amount,
```

Comment:
Minor observation, this parameter is named "underlyingAmount" in LoanRouter.sol

Reply:
fixed! thanks nice catch